### ISSUE TITLE
what does mean logits shape?

### ISSUE THREAD

**Issue by moonnyeon** [ethanfetaya/NRI#38]
what does mean logits shape ?
logits = encoder(pts, rel_rec, rel_send)
## my pts ----> torch.Size([32, 14, 30, 3])

logits - torch.Size([32, 182, 3])
